tequilapi/contract/session.go

```diff
@@ -22,34 +22,34 @@ import (
 	"time"
 
 	"github.com/mysteriumnetwork/node/consumer/session"
-	"github.com/vcraescu/go-paginator"
+	"github.com/mysteriumnetwork/node/tequilapi/utils"
 )
 
 // NewSessionListResponse maps to API session list.
 func NewSessionListResponse(
 	sessions []session.History,
-	paginator *paginator.Paginator,
+	paginator *utils.Paginator,
 	stats session.Stats,
 	statsDaily map[time.Time]session.Stats,
-) ListSessionsResponse {
+) SessionListResponse {
 	dtoArray := make([]SessionDTO, len(sessions))
 	for i, se := range sessions {
 		dtoArray[i] = NewSessionDTO(se)
 	}
 
-	return ListSessionsResponse{
+	return SessionListResponse{
 		Sessions:   dtoArray,
-		Paging:     NewPagingDTO(paginator),
+		Pagination: NewPaginationDTO(paginator),
 		Stats:      NewSessionStatsDTO(stats),
 		StatsDaily: NewSessionStatsDailyDTO(statsDaily),
 	}
 }
 
-// ListSessionsResponse defines session list representable as json.
-// swagger:model ListSessionsResponse
-type ListSessionsResponse struct {
+// SessionListResponse defines session list representable as json.
+// swagger:model SessionListResponse
+type SessionListResponse struct {
 	Sessions   []SessionDTO               `json:"sessions"`
-	Paging     PagingDTO                  `json:"paging"`
+	Pagination PaginationDTO              `json:"pagination"`
```

Comment:
Pagination fields should be at the root level and collection entries should be under a common name, e.g. `entries` or `items` so that it's easier for the client to consume.

Reply:
Fixed